### PR TITLE
feat(status-page): First PR for a Status Page

### PR DESCRIPTION
**Pull request by cyberhorsey**
Status Page that displays information about our protocol and network.

### CODE REVIEW

File: packages/status-page/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Bridge UI
```
Comment (d1onys1us) [inline packages/status-page/README.md:1]:
```suggestion
# Status page
```

**Comment by d1onys1us** [comment]
nit: pr title can be "status-page" instead of "status" for the scope. Eventually, will make the scopes required to match a discrete set (which is at minimum the names defined in package.json), and otherwise fail. Will help w auto changelog generation.

**Review by dionysuzx — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Comment by dantaik** [comment]
@cyberhorsey could you also deploy this on Vercel and config the workflow for auto-deployment?

**Comment by cyberhorsey** [comment]
> 

Yes of course, best to merge it to `main` first as Vercel deploys from `main` branch traditionally.

Can you configure a URL for us to point to Vercel, as I dont have domain control? `status.taiko.xyz` or something?

**Pull request merged**